Dockerfile-fpm

```diff
@@ -1,6 +1,13 @@
 FROM php:7.2-fpm-alpine3.7 as fpm
 
 ENV FCGI_CONNECT=/var/run/php-fpm.sock
+ENV PHP_FPM_PM=dynamic
```

Comment:
these defaults have been taken from
`docker run -it --rm php:7.3-fpm-alpine3.9 cat /usr/local/etc/php-fpm.d/www.conf`

Reply:
Perfect, that's what we want to avoid breaking compatibility for others!